src/features/restaurants/Dashboard.tsx

```diff
@@ -47,21 +47,37 @@ export const Dashboard = () => {
     content = <p>Loading...</p>
   } else if (status === 'succeeded') {
     content = (
-      <div className='w-100 d-flex flex-column align-items-center'>
-        <NavMenu />
-        <Button variant="primary" onClick={handleShow} className="m-2 align-self-start ">
+      <div
+        style={{
+          maxWidth: '1280px',
+          paddingTop: '4rem',
+        }}
+      >
+        <Button
+          variant="green-accent"
+          onClick={handleShow}
+          className="me-2 m-4"
+        >
           Filter
         </Button>
+        <NavMenu />
         <Map />
         <RestaurantList />
         <Filter show={show} handleClose={handleClose} />
       </div>
     )
   } else if (status === 'failed') {
     content = (
-      <Error message='There was a problem on our end. Please try again.'/>
+      <Error message="There was a problem on our end. Please try again." />
     )
   }
 
-  return <section className='w-100'>{content}</section>
+  return (
+    <div
+      className="fade-in w-100 d-flex flex-column align-items-center"
+      style={{ display: 'flex', justifyContent: 'center' }}
```

Comment:
I think we can get rid of lines 77 & 78, but move line 77 up to the div on line 50.